packages/vue-components/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@wmde/wikit-vue-components",
+  "version": "0.0.0",
+  "description": "The component implementation of wikit in vue",
+  "author": "The Wikidata team",
+  "homepage": "https://github.com/wmde/wikit#readme",
+  "license": "BSD-3-Clause",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wmde/wikit.git"
+  },
+  "scripts": {
+    "test": "echo \"Tests of vue-components\""
+  },
+  "bugs": {
+    "url": "https://github.com/wmde/wikit/issues"
+  },
+  "dependencies": {
+    "@wmde/wikit-tokens": "file:../tokens"
```

Comment:
This makes my head hurt a little. The difference between depending on a published npm package and pointing to the source of the package as a file dependency as it's done here is that the published package will of course contain only what's specified in the `files` field in its `package.json`, whereas the file dependency contains whatever the directory contains. This means that this sub package isn't self sufficient in that you would always first have to build the `tokens/` package or otherwise the components package would be missing the build output.

That said, I guess that's the whole point of lerna? The sub package doesn't need to be self-sufficient and can be interdependent because I'm not supposed to install/build/publish anything from here and instead do it from the root?